src/compiler/scala/tools/nsc/Properties.scala

```diff
@@ -13,7 +13,7 @@ object Properties extends scala.util.PropertiesTrait {
 
   // settings based on jar properties, falling back to System prefixed by "scala."
   def residentPromptString = scalaPropOrElse("resident.prompt", "\nnsc> ")
-  def shellPromptString    = scalaPropOrElse("shell.prompt", "\nscala> ")
+  def shellPromptString    = "\n" + Properties.propOrElse("scala.repl.PS1", "scala> ")
```

Comment:
I have created another branch topic/repl-customization--2nd-try. 
Let me know your opinion about it. 
Thank you.